language-plutus-core/test/Evaluation/ApplyBuiltinName.hs

```diff
@@ -59,109 +59,91 @@ test_typedAddInteger :: TestTree
 test_typedAddInteger
     = testProperty "typedAddInteger"
     $ prop_applyBuiltinName typedAddInteger (+)
-    $ genTypedBuiltinDef
 
 test_typedSubtractInteger :: TestTree
 test_typedSubtractInteger
     = testProperty "typedSubtractInteger"
     $ prop_applyBuiltinName typedSubtractInteger (-)
-    $ genTypedBuiltinDef
 
 test_typedMultiplyInteger :: TestTree
 test_typedMultiplyInteger
     = testProperty "typedMultiplyInteger"
     $ prop_applyBuiltinName typedMultiplyInteger (*)
-    $ genTypedBuiltinDef
 
 test_typedDivideInteger :: TestTree
 test_typedDivideInteger
     = testProperty "typedDivideInteger"
     $ prop_applyBuiltinName typedDivideInteger (nonZeroArg div)
-    $ genTypedBuiltinDivide
```

Comment:
We can actually use `genTypedBuiltinDef` everywhere and just ditch all other generators (of which we had only `genTypedBuiltinDivide` up until now, because all others were removed long time ago).